docs/source/quickstart.rst

```diff
@@ -0,0 +1,64 @@
+Quickstart
+==========
+
+Setup and installation
+----------------------
+
+There are two main steps to get up and running:
+
+Setup environment variables
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+AA Toolbox downloads data to the directory referenced by the
+environment variable `AA_DATA_DIR`. Before beginning, please make
+sure that this environment variable points to where you would
+like the data to go.
+
+Install AA Toolbox
+^^^^^^^^^^^^^^^^^^
+
+AA Toolbox supports Python 3.6 and newer. It can be installed in your Python
+environment using the following command:
+
+.. code-block:: sh
+
+    $ pip install aa-toolbox
+
+AA Toolbox should now be installed.
+
```

Comment:
I'm not sure if I would include it in quickstart because I would consider it more advanced usage. I did add some config docs in the custom config PR which I'm about to merge in, let me know if you think it's enough.

Reply:
Yes, looks great